### PR TITLE
Support sparse api in pir

### PR DESCRIPTION
**Pull request by risemeup1**
### PR Category

Others

### PR Types

Others
### Description

推全sparse python api
Pcard-67164

### CODE REVIEW

File: test/dygraph_to_static/test_tensor_attr_consistency.py [hunk 1]
```diff
@@ -76,12 +76,10 @@
         'set_vocab',
         'strides',
         'to',
-        'to_dense',
         'to_sparse_coo',
         'to_sparse_csr',
         'tolist',
         'value',
-        'values',
         'zero_',
```
Comment (SigureMo) [inline test/dygraph_to_static/test_tensor_attr_consistency.py:82]: `indices` 在 52 行～

**Review by chen2016013 — DISMISSED** [review]
Great Work!

**Review by zhangbo9674 — DISMISSED** [review]

File: test/dygraph_to_static/test_tensor_attr_consistency.py (hunk 1 above)
Comment (risemeup1) [reply test/dygraph_to_static/test_tensor_attr_consistency.py:82]: 没看到哈哈，下个PR删除掉

**Review by SigureMo — DISMISSED** [review]
LGTMeow  for update `test/dygraph_to_static/test_tensor_attr_consistency.py`

**Review by chen2016013 — APPROVED** [review]

**Review by SigureMo — APPROVED** [review]

**Pull request merged**